crates/pop-parachains/src/templates.rs

```diff
@@ -356,28 +311,36 @@ mod tests {
 	#[test]
 	fn test_network_config() {
 		let network_configs = template_network_configs();
-		for template in Template::VARIANTS {
+		for template in ParachainTemplate::VARIANTS {
 			assert_eq!(template.network_config(), network_configs[template]);
 		}
 	}
 
 	#[test]
 	fn test_default_template_of_provider() {
 		let mut provider = Provider::Pop;
-		assert_eq!(provider.default_template(), Template::Standard);
+		assert_eq!(provider.default_template(), ParachainTemplate::Standard);
 		provider = Provider::Parity;
-		assert_eq!(provider.default_template(), Template::ParityContracts);
+		assert_eq!(provider.default_template(), ParachainTemplate::ParityContracts);
 	}
 
 	#[test]
 	fn test_templates_of_provider() {
 		let mut provider = Provider::Pop;
 		assert_eq!(
 			provider.templates(),
-			[&Template::Standard, &Template::Assets, &Template::Contracts, &Template::EVM]
+			[
```

Comment:
Can this function not return owned values so we dont have to borrow for comparison?

Reply:
I did this, but then reverted the change. The reason is that it introduces unnecessary clones in its usage.